test/unit/specs/mount/options/slots.spec.js

```diff
@@ -25,6 +26,18 @@ describe('mount.slots', () => {
     expect(wrapper.contains('span')).to.equal(true)
   })
 
+  it('mounts component with default slot if passed string in slot object', () => {
+    const vm = new Vue()
+    if ('_v' in vm) {
```

Comment:
Could you find which Vue version supports _v and then import { vueVersion } from ~resources/test-utils. 

So instead of this:
```js
if ('_v' in vm) {
```
use this:
```js
if (vueVersion > 2.2) {
```
That way it makes more sense from other developers.

Reply:
https://github.com/vuejs/vue/blob/68abdc3c3c86e01089c32b434e662dc15a3a9eda/src/core/instance/render.js

[Vue.js 2.1.5](https://github.com/vuejs/vue/releases/tag/v2.1.5) supports  `vue._v()`.

Reply:
https://github.com/vuejs/vue/commit/79e1058799dcde0b7e341ef44b590ecbf1e3a722#diff-5b337525bf69cfea5590728194e8d92f

[Vue.js 2.1.5](https://github.com/vuejs/vue/releases/tag/v2.1.5) supports  `vue._v()`.

I will change this.

Reply:
I changed this.